packages/ui/src/components/UploadIds/UploadModal/index.tsx

```diff
@@ -96,119 +100,150 @@ const UploadModal = ({
         // eslint-disable-next-line
     }, [debouncedValue]);
 
+    const isMaximumItemsReached = () => {
+        if (limitItem) {
+            return getRawValueList().length > limitItem ? true : false;
```

Comment:
Le ternaire est pas nécessaire ici. Tu peux seulement laisser le 
```suggestion
            return getRawValueList().length > limitItem;
```